package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bellese/angular-design-system",
-  "version": "4.3.0",
+  "version": "4.4.0",
```

Comment:
did you make this change on purpose? I think the semantic versioning release bot will take care of this

Reply:
Confirmed, reverting to `4.3.0`